javascript/computer_science/common_data_structures_algorithms.md

```diff
@@ -52,6 +52,7 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
+- [LeetCode](https://leetcode.com/) - platform to practice on Data Structures and Algorithms (DSA). Recommended study from the section Explore.
```

Comment:
I think there might be a reason why we aren't already linking to leet code in these lessons. I'll raise it with the team and get back to you.

Reply:
Yep, we've decided not to include leet code here recentley. More context: https://github.com/TheOdinProject/curriculum/pull/26013#issuecomment-1660223050

We already link to it in our Getting hired course, with leet code being synonymous with interviews and to avoid any rabbit holds we think thats a better place for it.